.travis.yml

```diff
@@ -79,7 +79,7 @@ script:
 
 after_success:
   - if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_TAG" != "" ] && [ "$TEST_TARGET" == "default" ]; then
-      conda install -n root conda-build anaconda-client ;
+      conda install -n root conda-build=3 conda=4.4 anaconda-client ;
```

Comment:
It sucks that we need to pin those but it did work, see https://travis-ci.org/ocefpaf/pocean-core/jobs/320398907 where I force the conda-build block to be executed.

I'll investigate this and update the packages in conda-forge to avoid that bad match of conda/conda-build.